src/Servers/HttpSys/src/NativeMethods.txt

```diff
@@ -42,6 +42,7 @@ HTTPAPI_VERSION
 HttpCancelHttpRequest
 HttpCloseServerSession
 HttpCloseUrlGroup
+HttpCloseRequestQueue
```

Comment:
What's this file for?

Reply:
It's how the cswin32 source generator decides what methods to create PInvokes for
https://microsoft.github.io/CsWin32/docs/getting-started.html